Dockerfile

```diff
@@ -124,7 +125,7 @@ LABEL description="The authentication client required to expose secrets from a C
 
 # =================== CONTAINER FOR HELM TEST ===================
 
-FROM golang:alpine3.17 as k8s-cluster-test
+FROM golang:1.19-alpine as k8s-cluster-test
```

Comment:
For the scan error, we can go with `golang:alpine`

Also below please remove lines 147 - 153,
and uncomment lines 154 and 155.
That should fix the scan error.